spyder/utils/qthelpers.py

```diff
@@ -598,6 +598,31 @@ def create_plugin_layout(tools_layout, main_widget=None):
     return layout
 
 
+class MenuProxyStyle(QProxyStyle):
+    """Menu proxy class to right justify shortcuts in QMenu."""
+
+    def drawControl(self, element, option, painter, widget=None):
+        """Reimplement Qt method."""
+        shortcut = ""
+        if element == QStyle.CE_MenuItem:
+            vals = option.text.split("\t")
+
+            if len(vals) == 2:
+                text, shortcut = vals
+                option.text = text
+        super(MenuProxyStyle, self).drawControl(element, option, painter,
+                                                widget)
+        if shortcut:
+            margin = 10  # Margin from the right of menu column to shortcuts.

```

Comment:
This works as a constant, so I would probably moved this out of the method and make it a class variable

```
class MenuProxyStyle(QProxyStyle):
    RIGHT_MARGIN = 10  # Margin from the right of menu column to shortcuts.
```
It might help to add the units? I am guessing pixels/px?